app/presenters/publishing_api/editionable_worldwide_organisation_presenter.rb

```diff
@@ -23,9 +25,10 @@ def content
         details: {
           logo: {
             crest: "single-identity",
+            formatted_title: worldwide_organisation_logo_name(item),
           },
         },
-        document_type: item.class.name.underscore,
+        document_type: "worldwide_organisation",
```

Comment:
Is this okay? I was in the middle of changing Publishing API, and I wondering if we actually needed these to be different? We can distinguish the old ones from new through the base path instead?

Reply:
We could revert this if this is okay https://github.com/alphagov/publishing-api/pull/2578

If not, happy to add in the `editionable_worldwide_organisation` to the schema types.

Reply:
It's probably fine to keep both editionable and non-editionable as having the same document type, since the same code will be rendering both versions.